woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1973,6 +1973,26 @@ def test_accessor_drop(sample_df):
     assert to_pandas(multiple_list_df).equals(to_pandas(different_order_df))
 
 
+def test_accessor_drop_inplace(sample_df):
+    inplace_df = sample_df.copy()
+    inplace_df.ww.init()
+    schema_df = inplace_df.copy()
```

Comment:
The assertions here are mixing the use of `sample_df` and `schema_df`. I think we can delete this line and just use `sample_df` everywhere?

Reply:
Agreed! Done!